modules/cert-manager-override-scheduling.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-override-scheduling_{context}"]
+= Configuring scheduling overrides for cert-manager Operator components
+
+After installing the {cert-manager-operator}, you can configure the pod scheduling from the {cert-manager-operator} API for the cert-manager components such as cert-manager controller, CA injector, and Webhook. Configuring the `overrideScheduling` block in the `certmanager.operator` custom resource enables you to define `nodeSelector` and `tolerations` for the `controllerConfig`, `webhookConfig`, and `cainjectorConfig` sections respectively.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have installed version 1.15.0 or later of the {cert-manager-operator}.
+
+.Procedure
+
+* Update the `certmanager.operator` custom resource to configure pod scheduling overrides for the desired components. Use the `overrideScheduling` field under `controllerConfig`, `webhookConfig`, or `cainjectorConfig` to define `nodeSelector` and `tolerations` settings.
++
+.Example output
+[source,yaml]
+----
+$ oc patch certmanager.operator cluster --type=merge -p="
+spec:
+  controllerConfig:
+    overrideScheduling:
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: '' <1>
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule <2>
+  webhookConfig:
+    overrideScheduling:
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: '' <3>
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule <4>
+  cainjectorConfig:
+    overrideScheduling:
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: '' <5>
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule <6>
+----
+<1> Defines the control-plane `nodeSelection` for the `controllerConfig` override scheduling.
+<2> Defines the `toleration` for `controllerConfig`.
+<3> Defines the `nodeSelector` for `webhookConfig`.
+<4> Defines the `toleration` for `webhookConfig`.
+<5> Defines the `nodeSelector` for `cainjectorConfig`.
+<6> Defines the `toleration` for `cainjectorConfig`.
+
+
+.Verification
+
+. Verify the updated `certmanager.operator` custom resource by:
```

Comment:
We're not actually checking the `certmanager.operator` resource in this verification stage,
please rephrase this statement.

`Verify the pod scheduling changes have been applied` ? or something similar generic would help.